barcampgb.org

```diff
@@ -0,0 +1 @@
+Subproject commit 8ae7da9b8bfbfc64e32d7d83f9d693bc1d606f42
```

Comment:
I cloned your repo and it appears as another directory called `barcampgb.org`. Seems like somehow this repo got included as a submodule, but the mapping isn't complete. You should be able to remove it by running `rm -r barcampgb.org` from command line.